catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/CswEndpoint.java

```diff
@@ -713,43 +725,54 @@ private int updateRecords(UpdateAction updateAction)
             queryRequest = queryFactory.updateQueryRequestTags(queryRequest,
                     schemaTransformerManager.getTransformerSchemaForId(updateAction.getTypeName()));
 
-            QueryResponse response = framework.query(queryRequest);
+            Map<String, Serializable> recordProperties = updateAction.getRecordProperties();
+            Iterable<List<Result>> resultList = Iterables.partition(ResultIterable.resultIterable(
+                    framework,
+                    queryRequest), 64);
```

Comment:
I'm guessing on an appropriate size that will definitely not trigger an OOM condition, but I agree that it's probably lower than strictly necessary. @kcwire?